docs: Replicates changes made to Infura tutorials (#1470)
MetaMask/metamask-docs: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md

## Steps

### 1. Select your network and verify funds

<Tabs>
  <TabItem value="View" label="Sepolia" default>

> Verify that your wallet has testnet ETH for the Sepolia network. You may use the [Infura faucet](https://www.infura.io/faucet) to top up. 

Reviewer: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may

```suggestion
Verify that your wallet has testnet ETH for the Sepolia network. Use the [Infura faucet](https://www.infura.io/faucet) to add funds. 
```

</TabItem>
<TabItem value="Retrieve" label="Alternative network" default>

To use an alternative network:
- Update your [environment file](#4-create-the-env-file) with the alternative network name 
- Have testnet ETH for that network
</TabItem>
</Tabs>

### 2. Create a project directory

> Create a new directory for your project. This can be done from the command line:

Reviewer: ```suggestion
Create a new directory for your project:
```

or  

```suggestion
Create a new directory for your project using the command line:
```

<Tabs>
  <TabItem value="View" label="Syntax" default>

Replace the following values in the `.env` file:

```bash
ETHEREUM_NETWORK = "<network>"
INFURA_API_KEY = "<API-KEY>"
SIGNER_PRIVATE_KEY = "<PRIVATE-KEY>"
```

- `<NETWORK>` with "sepolia" or the network you are using.
- `<API-KEY>` with your API key of the web3 project.
- `<PRIVATE-KEY>` with the [private key of your Ethereum account](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-Export-an-Account-Private-Key). A transaction must be signed with the sender's private key. Make sure that you prefix the `SIGNER_PRIVATE_KEY` value with `0x`. The private key you export from MetaMask isn't prefixed with `0x`.

</TabItem>
<TabItem value="Retrieve" label="Example" default>

```text
ETHEREUM_NETWORK = "sepolia"
INFURA_API_KEY = "d23...x...6e"
SIGNER_PRIVATE_KEY = "0x561...x...61df"
```
</TabItem>
</Tabs>

:::danger

Never disclose your private key. Anyone with your private keys can steal the assets controlled by those keys.

:::

### 5. Create `eip1559_tx.js` file

Create a file `eip1559_tx.js` in the project directory which configures and sends the transaction. For example:

:::info

> Optional: to send test ETH to an account of your choice, update line 15 with your selected account.

Reviewer: I think _optional_  can be removed.

```suggestion
To send test ETH to a specific account, update line 15 with the desired account address.
```

:::

```javascript showLineNumbers

> const { ethers, parseUnits } = require("ethers");

Reviewer: I think code should be formatted for 2 space indentation (for consistency)

    async function main() {
      // Configuring the connection to an Ethereum node
      const network = process.env.ETHEREUM_NETWORK;
      const provider = new ethers.InfuraProvider(
        network,
        process.env.INFURA_API_KEY
      );
      // Creating a signing account from a private key
      const signer = new ethers.Wallet(process.env.SIGNER_PRIVATE_KEY).connect(provider);

      // Creating and sending the transaction object
      const tx = await signer.sendTransaction({
        to: "0x618917c657e9F5b346c0141CB14F5D3CED65D449", // Replace with your selected account
        value: parseUnits("0.001", "ether"),
      });
      console.log("Mining transaction...");
      console.log(`https://${network}.etherscan.io/tx/${tx.hash}`);
      // Waiting for the transaction to be mined
      const receipt = await tx.wait();

>       // The transaction is now on chain!

Reviewer: ```suggestion
      // The transaction is now on chain
```
